### PR TITLE
project-15-shaun-newmark

### PR DESCRIPTION
**Pull request by aicb1**

### CODE REVIEW

**Review by johlofg — APPROVED** [review]
Nice project! 
I don´t have that much to add. 
In css you mix some hex, rgb and name for colors otherwise good job. 
Tested the app and it works as it should and at least all blue goals are reached. 
Your code is clear and easy to read.  Good job!

File: code/src/index.css [hunk 1]
```diff
@@ -11,3 +18,40 @@ code {
...
+  display: flex;
+  flex-direction: column;
+  margin: 10px;
+  background: linear-gradient(
+    189deg,
+    rgba(166, 228, 255, 1) 0%,
+    rgba(125, 113, 195, 1) 72%
+  );
+  padding: 30px;
+  width: auto;
```
Comment (KarolCislowski) [inline code/src/index.css:32]: It is not a good practice to do that. You're completely losing control over your container.
If you still want to use it as that you need to think about making your inputs stretch ass well, or move your content to the center.

File: code/src/index.css [hunk 2]
```diff
@@ -1,3 +1,10 @@
+html {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: black;
```
Comment (KarolCislowski) [inline code/src/index.css:5]: why you used `html` instead of `body` to set these styles?

File: code/src/components/TodoList.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      {items.map((todo) => (
+        <div key={todo.id} className="todo-item">
+          <button type="button" className="delete-btn" onClick={() => dispatch(todos.actions.removeTodo(todo.id))}>
+            X
+          </button>
+          <p>{todo.description}</p>
+          <input
+            type="checkbox"
+            checked={todo.isComplete}
+            onChange={() => dispatch(todos.actions.toggleComplete(todo.id))} />
```
Comment (KarolCislowski) [inline code/src/components/TodoList.js:21]: IMHO, Layout: checkbox, text, delete btn, wóuld be more intuitive. 
To make it even cleaner you can wrap checkbox and text with a container and kick your delete button by using `space-between`

**Review by KarolCislowski — APPROVED** [review]
Nice and clean logic. 
Remember that your projects are your ticket to a new career so make them look as nice as possible.

**Pull request closed without merging**